packages/story-editor/src/app/media/utils/useMediaUploadQueue/index.js

```diff
@@ -386,6 +386,72 @@ function useMediaUploadQueue() {
         isTranscoding: state.queue.some((item) => item.state === 'TRANSCODING'),
         isMuting: state.queue.some((item) => item.state === 'MUTING'),
         isTrimming: state.queue.some((item) => item.state === 'TRIMMING'),
+        isResourceProcessing: (resourceId) =>
```

Comment:
Some inline documention / JSDoc blocks for these new functions would go a long way to help explain the differences.